meta-luneos/recipes-connectivity/mobile-broadband-provider-info/mobile-broadband-provider-info_git.bbappend

```diff
@@ -1,3 +1,5 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
+
 #Mer uses their own version of MBPI which has some elements added. So we want to use their version only in case we're using their oFono version. In other cases where we use upstream oFono we want to use the regular MBPI from Yocto.
 
 SRC_URI:halium  = " \
```

Comment:
S:halium = "${WORKDIR}/git/${BPN}"

Reply:
Do you mean the prepend is useless if I fix the S:halium line?

Reply:
New version pushed, with BPN instead of PN

Reply:
No, the S issue is unrelated to this, I've just noticed when looking at the rest of bbappend when checking why FILESEXTRAPATHS was needed.

with PN in S it would fail with multilib or if ever building nativesdk version, so it's better to get used to BPN in S (as with FILESEXTRAPATHS).

Reply:
mmh I have a build configure error now with meson:
```
ERROR: Neither source directory '/media/LuneOS/testing/webos-ports/tmp-glibc/work/cortexa8t2hf-neon-halium-webos-linux-gnueabi/mobile-broadband-provider-info/20240407/git/mobile-broadband-provider-info' nor build directory '/media/LuneOS/testing/webos-ports/tmp-glibc/work/cortexa8t2hf-neon-halium-webos-linux-gnueabi/mobile-broadband-provider-info/20240407/build' contain a build file meson.build.
```
I'll investigate if it was ok before the change

Reply:
Ok, it was unrelated. Fixed in very latest push.